### PR TITLE
Branding updates, asset referencing scheme & Shoelace icons

### LINKED ISSUE #291: ReplayWebpage Branding Update
### Context

This issue has already become a little bigger than an asset swap.... TL;DR We have new branding now!  It's already rolled out in #278 and should also make its way into the app!

...But now that we're adding new assets, it might also be nice to overhaul how we address them in ReplayWebpage so that it's in-line with Browsertrix?  ~If we're going to deal with asset imports the same way, that means all of the font awesome icons imported as SVG code and not file paths _also_ have to change and so I figure it makes sense to transition to `<sl-icon>` at the same time.~

### Tasks
- [x] Finalize ReplayWebpage app icon for desktop & PWAs
- [x] ReplayWebpage in-app icon (for use with `sl-icon` both in Browsertrix and in the menu item)
- [x] Update favicons
- [x] Update the asset addressing system to use pathing with the TsconfigPathsPlugin
-  ~Relink SVGs now that importing them returns paths instead of SVG code~
    -  ~If I need to relink everything that is imported this way, it ALSO makes sense to use this time to switch to `sl-icon`~
- [x] Add the app logos under the new `src/assets/brand` folder, also matching Browsertrix!
- [x] Add static app icon for loading screen

### PR DESCRIPTION
**Pull request by Shrinks99**
Fixes #291 

### Changes
- Updates ReplayWebpage branding!
  - The new logo and type lockup is now used everywhere the old graphics were used
  - The embed now uses the ReplayWebpage logo!
  - Adds the new logo to README wrapped in an H1 with alt text.  This logo will change the text color based on the GitHub light / dark theme the user has selected.
  - Adds new app icons for macOS and Windows
  - Adds new favicons & web manifest app icons
- Removes `fa-icon` adds `sl-icon`
  - Partially addresses #237 — That issue should be re-scoped and should cover converting them to Shoelace icon buttons.
  - Removed the few imported bootstrap icons that were used in the embed
  - ~Changes icons to align with ReplayWebpage 2.0 mockups~
  - Moves nav bar icons to align with ReplayWebpage 2.0 mockups
- Changes how images are imported across the whole app!
  - Importing an SVG or WEBP image will now import it as a path to be used in an `` attribute.
  - Mirrors how we do things in Browsertrix! :)
- Updates text in various places
  - Adheres to new terminology
  - Removes outdated references to WBN archives
- Unhides the load button on smaller screen sizes
- Improves navigation at smaller screen sizes
  - Moves main navigation buttons from the dropdown back to the nav bar on mobile
  - Does not add the sidebar back to the main nav bar as this sends the user to the full page non-sidebar version of the navigator and it can be confusing to try and get back to the page you were looking at but have that option be unavailable.

### Caveats
- The logo is no longer animated on loading
  - I would like to address this one later with an SVG animation that only runs when loading actions are taking place to better emphasize when a loading action is occurring
- The app logo in the top left doesn't collapse to remove the text on mobile
  - Probably not that hard to add a switch between `brandLockupColor` and `rwpLogo` but I don't know if we have that available to us in JS?  Pls advise <3
- Doesn't update any of the brand colours outside of logos
  - Would like to standardize with Browsertrix once components get replaced, preferably from a unified design token library of values.
- Does not link to new docs domain that will be used for #250, should be added in a followup PR to dev 2.0 once that's in main

### Screenshots!

**New VS old mobile menus**
Screenshot 2024-04-01 at 5 59 07 PM

**App icon on the service worker broken page**
Screenshot 2024-04-01 at 4 30 02 PM

**App icons in the embed**
Screenshot 2024-04-01 at 6 01 01 PM

**New macOS app icon!**
Screenshot 2024-04-01 at 6 03 24 PM

### CODE REVIEW

**Comment by Shrinks99** [comment]
@ikreymer Have reverted to Fontawesome and also updated to Fontawesome v6.

Screenshot 2024-04-04 at 4 21 55 PM

File: src/appmain.ts [hunk 1]
```diff
@@ -741,18 +713,15 @@ export class ReplayWebApp extends LitElement {
                       IS_APP
                         ? html`
                             <p>
-                              ReplayWeb.page App is a standalone app for Mac,
-                              Windows and Linux that loads web archive files
+                              ReplayWeb.page is a standalone app for Mac,
```
Comment (ikreymer) [inline src/appmain.ts:716]: This is less clear, 'archived items' is what we call things in our app, but what we are dealing with is allowing users to load web archive files, which we then refer to as archived items. In general, archived items could be anything, not related to web archives

File: src/appmain.ts [hunk 2]
```diff
@@ -741,18 +713,15 @@ export class ReplayWebApp extends LitElement {
...
                             </p>
                           `
                         : html` <p>
-                            <a href="https://replayweb.page" target="_blank"
-                              >ReplayWeb.page</a
-                            >
-                            is a browser-based viewer that loads web archive
-                            files provided by the user and renders them for
-                            replay in the browser.
+                            ReplayWeb.page is a browser-based viewer that loads
```
Comment (ikreymer) [inline src/appmain.ts:722]: Same as above, let's revert this change. The link is useful for opening a link to our site when using the embedded mode.
